### PR TITLE
Add docs for the new poolstation integration

### PR DESCRIPTION
**Pull request by cibernox**
## Proposed change
Adds documentation for the new Poolstation integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/55954
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2805
- This PR fixes or closes issue: None

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by cibernox** [comment]
@thecode rebased

**Comment by cibernox** [comment]
Brands PR has been merged!

File: source/_integrations/poolstation.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - Sensor
+ha_codeowners:
+  - '@cibernox'
+ha_domain: poolstation
+ha_platforms:
+  - sensor
+ha_quality_scale: silver
+---
+
+The Poolstation integration allows you to control and monitor your your pool equipment, like Idegis and AstralPool chlorinators, that connect to the [Poolstation](https://poolstation.net) cloud platform.
```
Comment (dgomes) [inline source/_integrations/poolstation.markdown]:
```suggestion
The Poolstation integration allows you to control and monitor your pool equipment, like Idegis and AstralPool chlorinators, that connect to the [Poolstation](https://poolstation.net) cloud platform.
```

File: source/_integrations/poolstation.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+It provides sensors for current PH level, temperature, electrolysis production and salt concentration.
```
Comment (dgomes) [inline source/_integrations/poolstation.markdown]:
```suggestion
It provides sensors for current pH level, temperature, electrolysis production and salt concentration.
```

**Comment by cibernox** [comment]
@dgomes good catch!

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**